From 4f992675aeff74281ff72b6f7b764c3f5240dfc6 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 15 Jan 2015 10:33:50 +0100
Subject: [PATCH] network/net-lib.sh:parse_iscsi_root(): fix c&p bug

fix iscsi_target_name for eui.* and naa.* target names

(cherry picked from commit 7cddd7b838941cc2608da8641a84031d696e18e0)
---
 modules.d/40network/net-lib.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules.d/40network/net-lib.sh b/modules.d/40network/net-lib.sh
index 82827b6..89b1101 100755
--- a/modules.d/40network/net-lib.sh
+++ b/modules.d/40network/net-lib.sh
@@ -318,12 +318,12 @@ parse_iscsi_root()
             v=${v%:iqn.*}:
             ;;
         *:eui.*)
-            iscsi_target_name=iqn.${v##*:eui.}
-            v=${v%:iqn.*}:
+            iscsi_target_name=eui.${v##*:eui.}
+            v=${v%:eui.*}:
             ;;
         *:naa.*)
-            iscsi_target_name=iqn.${v##*:naa.}
-            v=${v%:iqn.*}:
+            iscsi_target_name=naa.${v##*:naa.}
+            v=${v%:naa.*}:
             ;;
     esac
 
